packages/components/form/src/label-wrap.ts

```diff
@@ -78,18 +81,18 @@ export default defineComponent({
       )
     })
 
-    function render() {
+    return () => {
```

Comment:
Consider give the returned function a name here for better stack trace when debugging.

Reply:
I don't know how this would be good for debug, since most of our code uses arrow functions directly.